### PR TITLE
Content: Cats Watch Fox Hunt in Winter on Plains

### PR DESCRIPTION
**Pull request by Artyphex**
The patrol spots a fox leaping and diving into the snow. They either try it themselves, laugh, or snap at each other to focus.

### CODE REVIEW

File: resources/dicts/names/names.json
Comment (Thlumyn) [inline resources/dicts/names/names.json:1]: If you want to add more names, please make that a separate PR

File: resources/dicts/patrols/plains/hunting/leaf-bare.json [hunk 1]
```diff
@@ -1574,5 +1574,87 @@
...
+                "warrior": [3, 6],
+                "leader": [3,6],
+                "all apprentices": [3, 6]
+            } 
+        },
+        {"weight": 20,
+            "chance_of_success": 30,
+            "relationship_constraint": null,
+            "pl_skill_constraint": null,
+            "intro_text": "Leaf-bare is taking its toll. Your patrol is cold, frusterated, and most of all, hungry. r_c notices something odd. Is that a fox in the distance? Why is it jumping into the snow like that?",
```
Comment (Thlumyn) [inline resources/dicts/patrols/plains/hunting/leaf-bare.json]: spelling - "frustrated"

File: resources/dicts/patrols/plains/hunting/leaf-bare.json [hunk 2]
```diff
@@ -1574,5 +1574,87 @@
...
+            "decline_text": "First the cold, and the snow, and now a fox? This is too much for one day, p_l decides. Back to camp, another patrol can try again later. ",
```
Comment (Thlumyn) [inline resources/dicts/patrols/plains/hunting/leaf-bare.json:1608]: is this meant to be p_l or r_c ? This might be a different cat than the one that saw the fox

File: resources/dicts/patrols/plains/hunting/leaf-bare.json [hunk 3]
```diff
@@ -1574,5 +1574,87 @@
...
+            "success_outcomes": [
+                { 
+            "text": "s_c watches, copying how the fox walks close to the snow, ears down, imagining what it might be listening for until- that's it! s_c leaps into the air, diving headfirst into the snow. The patrol snickers as {PRONOUN/s_c/poss} rear flails above the snow, but that snickering turns to praise when s_c pops back up with a fat mouse between {PRONOUN/s_c/poss} teeth.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/plains/hunting/leaf-bare.json:1611]: again, which cat is this supposed to be? r_c or s_c ?

File: resources/dicts/patrols/plains/hunting/leaf-bare.json [hunk 4]
```diff
@@ -1574,5 +1574,87 @@
...
+            "relationships": [
+                {
+                    "cats_to": ["s_c"],
+                    "cats_from": ["patrol"],
+                    "mutual": false,
+                    "values": ["respect, trust, platonic"],
+                    "amount": 5
+                },
+            { 
+            "text": "The whole patrol stares at the sight of a fox, something cats fear, stuck upside with its whole rump flailing up and undignified. r_c can't stop the unattractive snort that leaves {PRONOUN/r_c/poss}, and then the whole patrol is laughing. For this one moment, your cats forget about the cold in the fur and hunger in their bellies to enjoy a moment with their friends.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/plains/hunting/leaf-bare.json]: {PRONOUN/r_c/poss} mouth? Their what?

**Review by Thlumyn — CHANGES_REQUESTED** [review]

File: resources/dicts/names/names.json
Comment (Artyphex) [reply resources/dicts/names/names.json:1]: Sorry about this! I didn't realize if I made another without branching first it would attach itself to the first PR. Won't happen again!

File: resources/dicts/patrols/plains/hunting/leaf-bare.json (hunk 2 above)
Comment (Artyphex) [reply resources/dicts/patrols/plains/hunting/leaf-bare.json:1608]: In my mind, I thought the implication was the patrol was speaking to each other so if the cats were different it wouldn't be confusing. Do you think it would be better if it were all just s_c?

File: resources/dicts/patrols/plains/hunting/leaf-bare.json (hunk 1 above)
Comment (Artyphex) [reply resources/dicts/patrols/plains/hunting/leaf-bare.json]: Tweaked in source file, will update soon.

File: resources/dicts/patrols/plains/hunting/leaf-bare.json (hunk 3 above)
Comment (Artyphex) [reply resources/dicts/patrols/plains/hunting/leaf-bare.json:1611]: Same as above! Do you think them being potentially different cats (r_c sees it and s_c reacts) is too confusing?

File: resources/dicts/patrols/plains/hunting/leaf-bare.json [hunk 5]
```diff
@@ -1574,5 +1574,87 @@
...
+            "relationships": [
+                {
+                    "cats_to": ["s_c"],
+                    "cats_from": ["patrol"],
+                    "mutual": false,
+                    "values": ["respect, trust, platonic"],
+                    "amount": 5
+                },
+            { 
+            "text": "The whole patrol stares at the sight of a fox, something cats fear, stuck upside with its whole rear flailing up and undignified. r_c can't stop (PRONOUN/r_c/self) from snorting at the sight. Then the whole patrol is laughing. For this one moment, your cats forget about the cold in the fur and hunger in their bellies to enjoy a moment with their friends.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/plains/hunting/leaf-bare.json]: Looks like this pronoun tag should have {} instead of ()

File: resources/dicts/names/names.json
Comment (Thlumyn) [inline resources/dicts/names/names.json:1]: The changes to names still need to be removed

**Review by Thlumyn — CHANGES_REQUESTED** [review]

**Comment by Thlumyn** [comment]
Just as a heads up, to make things easier you'll probably want to do your changes like this on a separate branch on your personal fork of this: like Artyphex/catfoxwinter or something like that, instead of Artyphex/development. It makes it easier to sync changes with the base clangen repository, and also easier to compare changes with the clangen/development branch (so its not, like, comparing "development" to "development").

**Comment by Artyphex** [comment]
> Just as a heads up, to make things easier you'll probably want to do your changes like this on a separate branch on your personal fork of this: like Artyphex/catfoxwinter or something like that, instead of Artyphex/development. It makes it easier to sync changes with the base clangen repository, and also easier to compare changes with the clangen/development branch (so its not, like, comparing "development" to "development").

Got it! I'll do that in the future. Still figuring Github out, I appreciate your patience!

**Comment by Artyphex** [comment]
I messed this PR up, closing it and reopening a fixed one
